extensions/notebook/src/jupyter/jupyterServerInstallation.ts

```diff
@@ -468,6 +468,9 @@ export class JupyterServerInstallation implements IJupyterServerInstallation {
 	 * Opens a dialog for configuring the installation path for the Notebook Python dependencies.
 	 */
 	public async promptForPythonInstall(kernelDisplayName: string): Promise<void> {
+		if (this._runningOnSAW) {
```

Comment:
So this is slightly better - but I still have to question why this is even necessary. Shouldn't isPythonInstalled and areRequiredPackagesInstalled be true below for saw builds where python is already pre-installed? Or is there a specific reason you want to avoid doing this check completely?